server/routes/conditions.ts

```diff
@@ -35,6 +36,41 @@ export default ({ licenceService, conditionsService }: { licenceService: Licence
       })
     }
 
+    router.post('/standard/:bookingId', audited, asyncMiddleware(postStandard))
+    router.post('/standard/:action/:bookingId', audited, asyncMiddleware(postStandard))
+
+    async function postStandard(req, res) {
+      logger.debug('POST /standard/')
+      const { bookingId, action } = req.params
+      const isChange = action === 'change'
+
+      await licenceService.update({
+        bookingId,
+        originalLicence: res.locals.licence,
+        config: formConfig.standard,
+        userInput: req.body,
+        licenceSection: 'licenceConditions',
+        formName: 'standard',
+        postRelease: res.locals.postRelease,
+      })
+
+      const conditionsVersion = res.locals.licence.versionDetails?.conditions_version
+
+      if (!conditionsVersion) {
+        await licenceService.setConditionsVersion(Number(bookingId), CURRENT_CONDITION_VERSION)
+      }
+
```

Comment:
Looks as if the conditions_version will be set to CURRENT_CONDITION_VERSION for every licence record seen by this handler.  Is that right?
Should this only happen if the record doesn't currently have an 'additional' section or if a new version of the licence is being created?  Not sure.

Reply:
I think we need a migration to set it if the licence has a `standard` form section. 

The section gets created whether or not an answer is selected.

So this would mean that we would set the condition to the default value if they've seen the conditions an clicked 
submit. 

I think we'll need another point where we set it though as users might not even click on this page.
The migration will need to respect that also